Detectors/TRD/workflow/io/src/TRDDigitReaderSpec.cxx

```diff
@@ -87,7 +87,7 @@ void TRDDigitReaderSpec::run(ProcessingContext& pc)
   pc.services().get<ControlService>().readyToQuit(QuitRequest::Me);
 }
 
-DataProcessorSpec getTRDDigitReaderSpec(int channels, bool useMC)
+DataProcessorSpec getTRDDigitReaderSpec(bool useMC)
```

Comment:
I wonder what the original `channels` argument was for?

Reply:
Maybe @jolopezl knows?

Reply:
@tdietel, @martenole: "maybe" is just some initial design that has never followed

Reply:
I think that was originally me, and had something to do with pipe-lining the dpl, but was not done.